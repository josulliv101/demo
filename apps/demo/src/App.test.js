import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import App from "./App";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("creates snapshot", () => {
  const wrapper = shallow(<App />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
