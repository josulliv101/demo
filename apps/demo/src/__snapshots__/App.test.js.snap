// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates snapshot 1`] = `
<BrowserRouter>
  <div
    className="App"
  >
    <WithStyles(Header) />
    <main
      className="App-header"
    >
      foobar
    </main>
    <withState(withHandlers(WithStyles(Footer))) />
  </div>
</BrowserRouter>
`;
